src/classes/issues-processor.ts

```diff
@@ -571,6 +578,10 @@ export class IssuesProcessor {
     if (this._shouldRemoveStaleWhenUpdated(issue) && issueHasComments) {
       await this._removeStaleLabel(issue, staleLabel);
 
+      // Are there labels to remove or add when an issue is no longer stale?
+      await this._removeLabelsWhenUpdatedFromStale(issue, removeLabelsWhenUpdatedFromStale);
```

Comment:
It's nice to defer the logic to reduce the complexity of this file.
One thing that could maybe be even better is to completely defer it to a dedicated class/service like milestones and assignees works.